specs/agents/tracing-instrumentation-db.md

```diff
@@ -188,6 +188,23 @@ If they don't, they MUST use a hard-coded list of URLs that correspond with the
 |`_.type`| `mongodb` | |
 |`_.name`| e.g. `customers` | Database name, same as `db.instance` if available  |
 
+#### `mongodb_capture_statement_commands` configuration
+
+Agents that support capturing MongoDB statements SHOULD implement this option.
```

Comment:
To reduce the risk of capturing sensitive information, I'd argue that all agents must implement the option with the specified defaults.

```suggestion
Agents that support capturing MongoDB statements MUST implement this option.
```